src/System.Threading/src/project.json

```diff
@@ -5,13 +5,7 @@
         "Microsoft.TargetingPack.Private.CoreCLR": "1.2.0-beta-24609-02"
       }
     },
-    "netcore50": {
-      "dependencies": {
-        "Microsoft.TargetingPack.Private.NETNative": "1.1.0-beta-24604-01",
-        "System.Runtime": "4.0.20"
-      }
-    },
-    "net46": {
+    "net463": {
       "dependencies": {
         "Microsoft.TargetingPack.NETFramework.v4.6": "1.0.1"
```

Comment:
>4.6 [](start = 47, length = 3)

4.6.2 -- I will stop commenting about these. Please take a look and update wherever needed.